ghostscript fails to build on aarch64

From https://flathub.org/builds/#/builders/45/builds/412, the build fails at

    configure: error: unrecognized option: `-DPNG_ARM_NEON=OFF'

@Alexander-Wilms You added this in e058f5aa60d87f28ec23dc49dd5741035eb1635a, presumably to fix something but the original build log is gone now. Maybe this should be `cppflags: -DPNG_ARM_NEON=OFF` instead of `config-opts`?

--- a/org.octave.Octave.yaml
+++ b/org.octave.Octave.yaml
@@ -105,8 +105,7 @@ modules:
     ldflags: -L/app/lib
     arch:
       aarch64:
-        config-opts:
-        - -DPNG_ARM_NEON=OFF
+        cppflags: -DPNG_ARM_NEON_OPT=0
   sources:
   - type: archive
     url: https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs926/ghostscript-9.26.tar.xz

